server/server.ts

```diff
@@ -51,7 +51,7 @@ if (featureSwitches.cspSecurityAudit) {
 	const csp = [
 		'report-uri /api/csp-audit-report-endpoint',
 		'report-to csp-endpoint',
-		`default-src ${cspDefaultSrcAllowList}`,
+		`default-src ${cspDefaultSrcAllowList}; default-style 'unsafe-inline'`,
```

Comment:
we are already joining on `; `
```suggestion
		`default-src ${cspDefaultSrcAllowList}`,
		`default-style 'unsafe-inline'`, // this is unsafe but needed for now for emotion
```

Reply:
I'm not sure how to deal with the single quotes properly though, is backticks the right way?

Reply:
also I added the comment in case someone in future is stealing our CSP policy :)